released/packages/coq-vcfloat/coq-vcfloat.2.1/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+version: "2.1"
```

Comment:
```suggestion
```
Version is defined by the filename path.

Reply:
done, in latest commit